### PR TITLE
Integrate new colour palette

### PR DESCRIPTION
**Pull request by urbanwide**
To view the palette and mappings:

[http://localhost:3000/admin/colours](http://localhost:3000/admin/colours)

### CODE REVIEW

File: app/assets/images/background-image.jpg
Comment (urbanwide) [inline app/assets/images/background-image.jpg:1]: This changes the dark blue to match the menu above

File: app/assets/stylesheets/redesign/nav.scss [hunk 1]
```diff
@@ -160,7 +160,7 @@ nav.navbar-second {
   }
 
   &.adult {
-    @include navbar($background: $blue-very-dark, $foreground: $white, $background-lozenge: $blue-medium, $foreground-lozenge: $blue-dark, $hover: $blue-medium);
+    @include navbar($background: $blue-very-dark, $foreground: $white, $background-lozenge: $blue-light, $foreground-lozenge: $blue-dark, $hover: $blue-light);
```
Comment (urbanwide) [inline app/assets/stylesheets/redesign/nav.scss:163]: Have switched the naming of the blues about in the palette slightly - see colours.rb for detail. The colour here is still the same as that being used before

**Review by ldodds — APPROVED** [review]
Looks good. Suggest merging this tomorrow so we can release on Thursday.

File: config/initializers/colours.rb [hunk 2]
```diff
@@ -12,8 +12,8 @@ module Colours
     palette: {
       blue: { # blue_pale etc
         pale: '#f2f6fc'.freeze,
-        light: '#cbe4fc'.freeze, # Blue medium from original design - used in nav. Blue light in original design is not distinct enough from this, so removed
-        medium: '#83a4df'.freeze, # new colour - derived (half way colour between light and dark) using https://www.dannybrien.com/middle/
+        light: '#dcecfd'.freeze, # did not use blue medium from original design as it is not distinct enough from light
```
Comment (urbanwide) [inline config/initializers/colours.rb:15]: Note the very slight change in the blue light (is now actually blue light from the original design, it was the blue medium before). This is the one I showed Claudia in the screen grabs as I think it looks marginally better in large blocks next to the other colours than the other one but in smaller detail, you'd not notice the change all. Have changed the derived new medium slightly too as it's the halfway point between blue light and bright.

**Comment by ldodds** [comment]
Good to merge

**Review by ldodds — APPROVED** [review]

**Pull request merged**